app/views/posts/show.html.erb

```diff
@@ -1,2 +1,13 @@
-<h1>Posts#show</h1>
-<p>Here is a single post for a given user</p>
+<section class="post_section">
+<div class="single_post_resume">
+<div id="post_header_div">
+<h3>Post by <%= @post.author.name %></h3>
+ <span>Comments: <%= @post.comments_counter %></span>
+ <span>Likes: <%= @post.likes_counter %></span>
+ </div>
+ <p><%= @post.text %></p>
+</div>
+</section>
+<section class="comment_section">
+<%= render partial:"post_comments", locals:{comments: @post.comments}%>
+</section>
```

Comment:
[OPTIONAL]
- [ ] Kindly find a way to display the comments, currently, they look like a database object.

![object](https://user-images.githubusercontent.com/53124548/199647949-15ac37df-b658-4aca-884f-84f3ec3e45c1.png)

Reply:
Hello, @elisha2kyakpo1
Thanks for your suggestion I will fix this.